events/events.go

```diff
@@ -49,3 +49,9 @@ type LookupEvent struct {
 	Key      string
 	Duration time.Duration
 }
+
+// Ready is fired when ringpop has successfully bootstrapped and is ready to receive requests and other method calls.
+type Ready struct{}
```

Comment:
I have been thinking about this, but since they are in the events package users would type something like `events.ReadyEvent` which I found too verbose.